plugins/continuous-toolbox/README.md

```diff
@@ -24,6 +24,7 @@ Note that this plugin uses APIs introduced in the `3.20200924.3` release of Bloc
 
 ```js
 import * as Blockly from 'blockly';
+import '../src/ContinuousCategory';
```

Comment:
So I'm pretty sure we only have this here because the ContinuousCategory registers itself (in the Blockly registry). But we shouldn't make the end user deal with that. I think it would work if we added this line in ContinuousToolbox instead, so that way when they include the toolbox, it will automatically include the category as well?

I'm pretty sure that should work, we can do it in the test directory and find out. (Feel free to ping me to do it if you want)

Reply:
Ok, so I think the problem is in `test/index.js` we are using the source files directly instead of using them through the `src/index.js` file. The `ContinuousCategory` is exported in there so if we use the `src/index.js` file we shouldn't have to include the `ContinuousCategory` anywhere else.

Reply:
Ok, good to know. Do we need to export the new class `ContinuousMetricsFlyout` in `src/index.js` as well? Right now it's only used internally, but someone might need to access it.